container/container_contract.go

```diff
@@ -582,6 +582,19 @@ func ListContainerSizes(epoch int) [][]byte {
 	return result
 }
 
+// IterateContainerSizes method returns iterator over container size estimations
+// that have been registered for the specified epoch.
+func IterateContainerSizes(epoch int) iterator.Iterator {
```

Comment:
Do we lose the information about the container in the result iterator? It seems just a key+size pair.

Reply:
Key is the key of a storage node, I doubt we can restore container ID from it.

Reply:
I think I don't understand the use-case. Will it be sufficient to just return total size for the epoch?
If not, what is done with the values obtained via this method?

Reply:
I think we should return both key and value, so container client will define iteration algorithm according to its needs.
As for total size, I agree that separate method will be sufficient.